week-1-practise/level-3/1-piping.js

```diff
@@ -16,16 +16,18 @@ var startingValue = 5;
 /* --------------------------- */
 
 function add(a, b) {
+  b = 10;
   return a + b;
```

Comment:
Not quite. You should not be reassigning a new value to an argument. Instead, you should provide that value _when you call the function_.
Same problem for `multiply` below. 
Can you think of how to do this?